beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -785,6 +785,10 @@ impl<T: BeaconChainTypes> Worker<T> {
 
                 verified_block
             }
+            Err(BlockError::PublishError) | Err(BlockError::SlashablePublish) => {
+                error!(self.log, "Gossip block triggered publish error");
```

Comment:
Would be nice to log the error here too

```suggestion
            Err(e @ BlockError::PublishError) | Err(e @ BlockError::SlashablePublish) => {
                error!(
                    self.log,
                    "Gossip block triggered publish error";
                    "error" => ?e
                );
```